ajax/libs/covervid/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "covervid",
+  "filename": "covervid.min.js",
+  "version": "1.0.0",
+  "description": "Make your HTML5 video behave like a background cover image with this lightweight Javascript plugin extension",
+  "author": "Stefan Erickson",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stefanerickson/covervid"
+  },
+  "keywords": [
+    "video",
+    "image",
+    "png"
+  ],
+  "npmName": "covervid",
```

Comment:
As same with @pvnr0082t, in npm(https://www.npmjs.com/package/covervid), the contributor is not the author of this library. So this library from npm is not officially maintained and we can't use it.
Because there is no git tag from GitHub repo, we can remove `"npmName"` and `"npmFileMap"`.